tests/unit/test_rest_streaming.py

```diff
@@ -97,30 +65,18 @@ def __init__(
         self._random_split = random_split
         self._response_message_cls = response_cls
 
-    def _parse_responses(self, responses: List[proto.Message]) -> bytes:
-        # json.dumps returns a string surrounded with quotes that need to be stripped
-        # in order to be an actual JSON.
-        json_responses = [
-            (
-                self._response_message_cls.to_json(r).strip('"')
-                if issubclass(self._response_message_cls, proto.Message)
-                else MessageToJson(r).strip('"')
-            )
-            for r in responses
-        ]
-        logging.info(f"Sending JSON stream: {json_responses}")
-        ret_val = "[{}]".format(",".join(json_responses))
-        return bytes(ret_val, "utf-8")
-
     def close(self):
         raise NotImplementedError()
 
     def iter_content(self, *args, **kwargs):
         return self._ResponseItr(
-            self._parse_responses(self._responses),
+            self._parse_responses(),
             random_split=self._random_split,
         )
 
+    def _parse_responses(self):
```

Comment:
nit: if you move this to the previous location of `_parse_responses` (ie above `def close`), it will make the diff more useful.

Reply:
addressed.